src/libecalc/presentation/yaml/mappers/utils.py

```diff
@@ -61,16 +58,10 @@ def resolve_reference(
 
 
 def resolve_and_validate_reference(value: str, references: Dict[str, ReferenceValue]) -> ReferenceValue:
```

Comment:
can we replace `resolve_reference` with `resolve_and_validate_reference` everywhere? no need to specify that we validate as well, it is just a natural thing of doing (in a submethod called `validate_reference` within `resolve_reference`?

Reply:
I have replaced everywhere, so yes. Should I rename it to resolve_reference?